src/foam/comics/v2/DAOControllerConfig.js

```diff
@@ -137,6 +137,42 @@ foam.CLASS({
           : of.getAxiomsByClass(foam.core.Property).map(p => p.name);
       }
     },
+    {
+      class: 'StringArray',
+      name: 'searchColumns',
+      factory: null,
+      expression: function(of, defaultColumns) {
+        var tableSearchColumns = of.getAxiomByName('searchColumns');
+
+        var filteredDefaultColumns = defaultColumns.filter(c => {
+          //  to account for nested columns like approver.legalName
+          if ( c.split('.').length > 1 ) return false;
```

Comment:
what would it take to search on nested columns?

Reply:
Would have to be an entirely new feature and doesn't seem efficient

Reply:
Filters right now work by adding a predicate to the dao table.  For references they can search by the  toSummary of the obj in the reference search view so that should  be fine already for  reference cases